code/_onclick/click.dm

```diff
@@ -285,6 +288,16 @@
 	if(SEND_SIGNAL(src, COMSIG_MOB_ATTACK_RANGED, A, modifiers) & COMPONENT_CANCEL_ATTACK_CHAIN)
 		return TRUE
 
+/**
+ * Ranged right click
+ *
+ * If the same conditions are met to trigger RangedAttack but it is
+ * instead initialized via a right click, this will trigger instead.
+ * Useful for mobs that have their abilities mapped to right click.
+ */
+/mob/proc/RangedRightClickOn(atom/A, modifiers)
+	if(SEND_SIGNAL(src, COMSIG_MOB_ATTACK_RANGED, A, modifiers) & COMPONENT_CANCEL_ATTACK_CHAIN)
```

Comment:
This doesn't look like it actually does anything, probably can afford to just leave this part blank.

`attack_secondary` and co use it because the original combat mode PR just attached right click stuff to the normal attacks, and I didn't feel like going back and fixing all of them.

I think I asked you to do this, use the signals, but I think it makes more sense to just leave it blank in hindsight.